lib/model.js

```diff
@@ -776,14 +776,14 @@ Model.prototype.model = function model(name) {
  * @api public
  */
 
-var CUSTOMIZABLE_DISCRIMINATOR_OPTIONS = {
-  toJSON: true,
-  toObject: true,
-  _id: true,
-  id: true
-};
-
 Model.discriminator = function discriminator(name, schema) {
+  var CUSTOMIZABLE_DISCRIMINATOR_OPTIONS = {
+    toJSON: true,
+    toObject: true,
+    _id: true,
+    id: true
+  };
+
   if (!(schema && schema.instanceOfSchema)) {
```

Comment:
see: http://mongoosejs.com/docs/api.html#model_
Moved CUSTOMIZABLE_DISCRIMINATOR_OPTIONS into discriminator() since it's the only place those options are used.